lib/Twig/SourceContextLoaderInterface.php

```diff
@@ -9,6 +9,13 @@
  * file that was distributed with this source code.
  */
 
+/**
+ * Adds a getSourceContext() method for loaders.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since 1.27 (to be removed in 3.0)
```

Comment:
But the `DebugClassLoader` will trigger it, won't it (in Symfony we also do not trigger deprecations manually in interfaces)? Most IDEs will display this as a misuse too.